conda-store-server/conda_store_server/alembic/versions/39ef0e9e43f6_add_tarball_ext_column_to_.py

```diff
@@ -0,0 +1,28 @@
+"""add tarball_ext column to CondaPackageBuild
+
+Revision ID: 39ef0e9e43f6
+Revises: 16f65805dc8f
+Create Date: 2023-02-08 13:19:52.799929
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '39ef0e9e43f6'
+down_revision = '16f65805dc8f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('conda_package_build', sa.Column('tarball_ext', sa.Unicode(length=64), nullable=True))
```

Comment:
never used `alembic` before but I'm smelling that this needs to check for existence... more later